### PR TITLE
Replace janky test suite with pg_regress

### PR DESCRIPTION
**Pull request by olirice**
## What kind of change does this PR introduce?
replaces the pytest test suite with pg_regress

Invoke with
```shell
./bin/installcheck
```

after installing dependencies (see README)

@w3b6x9 this is a test suite only change

The pytest suite was initially setup for running a couple of integration tests before I realized how many edge cases we'd want to cover. This change improves readability (see *.out files for explicitoutput) and extensibility

### CODE REVIEW

File: sql/walrus--0.1.sql [hunk 1]
```diff
@@ -194,7 +195,8 @@ $$;
 
 create type realtime.wal_column as (
     name text,
-    type text,
+    type_name text,
+    type_oid oid,
```
Comment (w3b6x9) [inline sql/walrus--0.1.sql]: @w3b6x9 Reminder to create the appropriate Realtime migration for these changes.

File: README.md [hunk 2]
```diff
@@ -339,13 +339,27 @@ The project is SQL only and can be installed by executing the contents of `sql/w
...
 
-pytest
+Install postgres
+```sh
+brew install postgres
+```
+
+Install wal2json
+```sh
+git clone https://github.com/eulerto/wal2json.git --depth 1 53b548a29ebd6119323b6eb2f6013d7c5fe807ec
```
Comment (w3b6x9) [inline README.md:354]: @olirice Can we drop the commit hash name or append to `wal2json-`?

```suggestion
git clone https://github.com/eulerto/wal2json.git --depth 1 wal2json-53b548a29ebd6119323b6eb2f6013d7c5fe807ec
```

Whatever we decide let's update the instruction on the next line (`cd wal2json`).

Comment (w3b6x9) [inline README.md:354]: @olirice Can we prepend `wal2json-` to the hash and shorten it?

```suggestion
git clone https://github.com/eulerto/wal2json.git --depth 1 wal2json-53b548a
```

Whatever we decide let's update the instruction on the next line (`cd wal2json`).

**Review by w3b6x9 — APPROVED** [review]
@olirice lgtm!

just one suggestion: https://github.com/supabase/walrus/pull/45#discussion_r853714189

File: README.md (hunk 2 above)
Comment (w3b6x9) [reply README.md:354]: Or `git clone https://github.com/eulerto/wal2json.git` then `git reset --hard 53b548a` to ensure new commits don't break the existing test suite.
